packages/vscode-extension-kogito-bundle/package.json

```
@@ -46,7 +46,8 @@
   "extensionDependencies": [
     "redhat.vscode-extension-bpmn-editor",
     "redhat.vscode-extension-dmn-editor",
-    "redhat.vscode-extension-serverless-workflow-editor"
+    "redhat.vscode-extension-serverless-workflow-editor",
+    "redhat.vscode-extension-dashbuilder"
```

```suggestion
    "redhat.vscode-extension-dashbuilder-editor"
```
Needs to be the same name defined in the `package.json` file.